tracer/src/Datadog.Trace/Ci/CiEnvironment/GitCommandGitInfoProvider.cs

```diff
@@ -0,0 +1,144 @@
+// <copyright file="GitCommandGitInfoProvider.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+using System.IO;
+using System.Linq;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.Ci.CiEnvironment;
+
+internal sealed class GitCommandGitInfoProvider : GitInfoProvider
+{
+    private GitCommandGitInfoProvider()
+    {
+    }
+
+    public static IGitInfoProvider Instance { get; } = new GitCommandGitInfoProvider();
+
+    private static DateTimeOffset UnixTimeStampToDateTime(long unixTimeStamp)
+    {
+        const long unixEpochTicks = TimeSpan.TicksPerDay * 719162; // 621,355,968,000,000,000
+        return new DateTimeOffset((unixTimeStamp * TimeSpan.TicksPerSecond) + unixEpochTicks, TimeSpan.Zero);
+    }
```

Comment:
`DateTimeOffset.FromUnixTimeMilliseconds` and `DateTimeOffset.FromUnixTimeSeconds` already exist, can't we use one of those? 🤔 Also, which is this doing? 😄

Reply:
🤦🏻 done in [6b0b8a9](https://github.com/DataDog/dd-trace-dotnet/pull/6259/commits/6b0b8a9ad9f70add2b7296b03b29ea484e47a859)